mysql-test/suite/rocksdb/t/i_s_ddl.test

```diff
@@ -17,6 +17,7 @@ CREATE TABLE is_ddl_t2 (x INT, y INT, z INT,
                         PRIMARY KEY (z, y) COMMENT 'zy_cf',
                         KEY (x)) ENGINE = ROCKSDB;
 
+--sorted_result
```

Comment:
Agreed, I can not recall the specific issue other than I got to learn about --sorted-result and something about it being preferred mtr semantics. I think ORDER BY is much more specific and clear to the intention. Will change it up and re-push to the PR.